src/pages/Home.jsx

```diff
@@ -1,7 +1,328 @@
+            {serviceDetails.map((service) => (
+              <div
+                key={service.id}
+                className="w-full md:w-60 p-5 flex flex-col justify-start md:justify-center items-start md:items-center gap-3 transition-all duration-300 hover:-translate-y-1 hover:shadow-lg cursor-pointer"
+              >
+                <p className="text-[#DB2777]">{service.icon}</p>
+                <h1 className="text-sm md:text-lg font-medium hover:text-[#DB2777]">
+                  {service.title}
+                </h1>
+              </div>
+            ))}
+          </div>
+        </div>
+
+        {/* Testimonials */}
+        <div
+          className={`bg-[#33313E] h-28 w-full rounded-lg overflow-hidden flex flex-row relative cursor-pointer}`}
+          onMouseEnter={() => setIsHoverd(true)}
+          onMouseLeave={() => setIsHoverd(false)}
+        >
+          <img
+            src={testimonial}
+            alt="TestimonialImage"
+            className="md:flex hidden object-cover w-1/2 h-full"
+          />
+          <div className="absolute right-8 md:right-40 mt-3 flex flex-col justify-center items-center">
+            <h1 className="text-white font-bold text-[20px] mb-2">
+              What Clients Says ?
+            </h1>
+            <p className="text-[13px] max-w-[300px] text-center text-[#FFB6C1]">
+              Real experiences, real beauty! See what our happy customers have
+              to say
+            </p>
+          </div>
+
+          {isHoverd && (
+            <div className="bg-[#33313E]/70 h-full w-full absolute z-50 flex justify-center items-center gap-3 cursor-pointer">
+              <h1 className="text-white font-bold text-[20px] text-center">
+                View Testimonials
+              </h1>
+              <ArrowRightCircle className="text-white" />
+            </div>
+          )}
+        </div>
+
+        {/* Common Questions Asked */}
+        <div className="flex flex-col gap-3 md:gap-10 mt-3 md:mt-10">
+          <h1 className="text-[25px] text-[#333333] font-bold text-start md:text-center">
+            Commonly Asked Questions
+          </h1>
+          <div className=" grid grid-cols-1 md:grid-cols-2 justify-between gap-5 md:gap-8 mt-3 ">
+            {commonlyQuestionAsked.map((quest, index) => (
+              <div key={index} className="flex flex-col gap-2">
+                <h1 className="text-[#DB2777] font-semibold text-[20px] ">
+                  {quest.Question}
+                </h1>
+                <p className="text-[#555555]">{quest.Answer}</p>
+              </div>
+            ))}
+          </div>
+        </div>
+
+        {/* Leave Comment */}
+        <div className="flex flex-col justify-start md:justify-center items-start md:items-center mt-8 gap-2">
+          <h1 className="text-[#333333] font-bold text-[20px]">
+            Feel free to leave comment☺️
+          </h1>
```

Comment:
`text-[20px]` is already known in tailwind, please do not use arbitrary values if a value already exist in tailwing